script.domoticz.scenes/.gitignore

```diff
@@ -0,0 +1,7 @@
+*.pyo
```

Comment:
This ha to be dropped from the submission

Reply:
fixed